pr_insight/algo/ai_handlers/openai_ai_handler.py

```
@@ -36,33 +37,32 @@ def deployment_id(self):
         """
         return get_settings().get("OPENAI.DEPLOYMENT_ID", None)
 
-    @retry(exceptions=(APIError, Timeout, TryAgain, AttributeError, RateLimitError),
+    @retry(exceptions=(APIError, Timeout, AttributeError, RateLimitError),
            tries=OPENAI_RETRIES, delay=2, backoff=2, jitter=(1, 3))
     async def chat_completion(self, model: str, system: str, user: str, temperature: float = 0.2):
         try:
             deployment_id = self.deployment_id
```

Ruff

Local variable `deployment_id` is assigned to but never used

(F841)